.github/workflows/ci.yml

```diff
@@ -112,6 +112,9 @@ jobs:
     - name: Clippy check
       run: cargo +${{ matrix.channel }} clippy --target ${{ matrix.target }} --features "${{ matrix.features }}" --verbose
 
+    - name: Clippy check zerocopy-derive
+      run: cargo +${{ matrix.channel }} clippy --manifest-path ./zerocopy-derive/Cargo.toml --target ${{ matrix.target }} --verbose
```

Comment:
Can you use the same logic from the "Run zerocopy-derive tests" step to skip when using features? In other words:

```yaml
      # Don't bother to check `zerocopy-derive` multiple times; that's what would
      # happen if we ran this step once for each set of `zerocopy` features.
      if: ${{ matrix.features == '' }}
```